hydromt_wflow/workflows/waterbodies.py

```diff
@@ -28,7 +28,7 @@ def waterbodymaps(
 
     The following waterbody maps are calculated:
 
-    - resareas/lakeareas : waterbody areas mask [ID]
+    - resareas/lake_area_id : waterbody areas mask [ID]
     - reslocs/lakelocs : waterbody outlets [ID]
```

Comment:
+ make sure the below is well implemented in the function
```suggestion
    - reservoir_area_id/lake_area_id : waterbody areas mask [ID]
    - reservoir_outlet_id/lake_outet_id : waterbody outlets [ID]
```